.github/workflows/update-submodules.yml

```diff
@@ -0,0 +1,42 @@
+name: Update Library Submodules
+
+on:
```

Comment:
Instead of having a manual workflow, can there be a periodic setup of the job for updating the submodules? (Like once every 24 hours if the submodule's commits have progressed)